backend-models/service-tests/courseedservice-test.txt

```diff
@@ -0,0 +1,1284 @@
+courseedservice.php
+
+---------------------------------------------------------------------------------------------------------------------
+
+========================================================================
+========================================================================
+===                           New Course                             ===
+========================================================================
+========================================================================
+
+*********
+* pre-req:
+********* 
+
+if(checklogin()){
+	$isSuperUserVar=isSuperUser($userid);
+
+	$ha = $isSuperUserVar;
+
+	if($ha){
+
```

Comment:
row 16 , 18, and 20:
You have just copied the code over to the text document. This does not give any information on what state the code needs to be in for the test to be able to executed.

What actually is a pre-req is: if(checklogin()): has to be true 
                                               $ha = value of super user

What actual values does $isSuperUserVar need to have for the test to proceed. it needs to be a super user. what value is associated with a super user?